ModularTegustation/tegu_items/representative/research/kcorp.dm

```diff
@@ -61,13 +61,28 @@
 	research_desc = "KCorp made this system for LCorp. <br>All you need to do is enter the following code into the machine, 9887. <br>Don't let LCorp know or they'll use it all the time and they'll break it; they have warranty."
 	corp = K_CORP_REP
 
-//Crit increaser
-/datum/data/lc13research/critincrease
-	research_name = "Repeatable: Crit Increaser Syringe "
-	research_desc = "Hey man, its Joe from research. <br>We got this new item here? Should stop you from going into critical condition; you just die. <br>Management says it needs to be tested, and you can buy more anytime."
+//Health booster
+/datum/data/lc13research/healthboost
+	research_name = "Repeatable: Health Booster Syringe "
+	research_desc = "Hey man, its Joe from research. <br>We've got a new item over here, something about increasing your overall health? <br>Management needs them tested, and what better place than here? You can buy more anytime."
 	cost = AVERAGE_RESEARCH_PRICE
 	corp = K_CORP_REP
 
-/datum/data/lc13research/critincrease/ResearchEffect(obj/structure/representative_console/caller)
-	new /obj/item/kcrit(get_turf(caller))
+/datum/data/lc13research/healthboost/ResearchEffect(obj/structure/representative_console/caller)
```

Comment:
After reading what it does, I realize I made a massive design mistake.
Making this repurchasable will lead to bullshit like having 700 HP.
Can you add a ..() in after this line to make it not repurchasable and change the description?